library/xmltraduit2.sax.po

```diff
@@ -0,0 +1,289 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2001-2016, Python Software Foundation
+# This file is distributed under the same license as the Python package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: Python 3.6\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2017-08-10 00:49+0200\n"
+"PO-Revision-Date: 2017-11-20 13:52+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: ../Doc/library/xml.sax.rst:2
+msgid ":mod:`xml.sax` --- Support for SAX2 parsers"
+msgstr ":mod:`xml.sax` --- Support SAX2 pour analyseurs"
+
+#: ../Doc/library/xml.sax.rst:11
+msgid "**Source code:** :source:`Lib/xml/sax/__init__.py`"
+msgstr "**Code source :** :source:`Lib/xml/sax/__init__.py`"
+
+#: ../Doc/library/xml.sax.rst:15
+msgid ""
+"The :mod:`xml.sax` package provides a number of modules which implement the "
+"Simple API for XML (SAX) interface for Python.  The package itself provides "
+"the SAX exceptions and the convenience functions which will be most used by "
+"users of the SAX API."
+msgstr ""
+"Le paquet :mod:'xml.sax' fournit des modules qui implementent l'interface "
+"Simple API for XML (SAX) pour Python. Le paquet en lui même fournit les "
```

Comment:
L'anglais, dans la traduction, on a tendance à le mettre en italique. L'italique dans le REsT s'écrit entre étoiles, comme ça :

    *Simple API for XML (SAX)*

Je te conseille d'aller lire notre README qui explique tous ces détails, je comprend que ce n'est pas évident de tous les retenir dès le début : https://github.com/python/python-docs-fr/tree/master